src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp

```diff
@@ -594,8 +594,8 @@ class boss_freya : public CreatureScript
                 const uint32 summonSpell[2][4] =
                 {
                               /* 0Elder, 1Elder, 2Elder, 3Elder */
-                    /* 10N */    {62950, 62953, 62955, 62957},
-                    /* 25N */    {62952, 62954, 62956, 62958}
+                    /* 10N */    {62957, 62955, 62953, 62950},
+                    /* 25N */    {62958, 62956, 62954, 62952}
```

Comment:
checked the gameobject template:
```code
ServerToClient: SMSG_GAMEOBJECT_QUERY_RESPONSE (0x0915) Length: 191 ConnIdx: 2 Time: 07/22/2012 22:49:10.632 Number: 388813
Entry: 194327
Type: Chest (3)
Display ID: 8628
[0] Name: Freya's Gift
[0] Data: 1634
[1] Data: 27081
[0] Quest Item: 45788 (45788)
```
the gameobject has freyas sigil as quest item, it is summoned by spell 62954, so it must be a hardmode chest (same for 62958 25man mode)
and checking more sniffs confirmed 3 elders are alive
i guess this will be the right order
```code
          /* 0Elder, 1Elder, 2Elder, 3Elder */
/* 10N */    {62950, 62952, 62953, 62954},
/* 25N */    {62955, 62956, 62957, 62958}
```
(had not the time to check all chests with sniffs)